test/commands/shell.js

```diff
@@ -71,6 +71,15 @@ commands.getprop = function getprop(name) {
   });
 };
 
+commands.chmod = function chmod(mode, file) {
+  mode = parseInt(mode, 10) || mode;
```

Comment:
Come to think of it, if we are just testing for digits, we don't need to parse to numbers at all. I'll update.